data/hai/hai reveal 4 middle.txt

```diff
@@ -1070,46 +1093,190 @@ mission "Hai Reveal [A09] History of Appeasement"
+			label questions
+			choice
+				# First phase of conversation: listen in or interrupt.
+				`	(Listen in on the conversation.)`
+					goto listen
+					to display
+						not "Hai Reveal [A09] History of Appeasement: label listen"
+						not "Hai Reveal [A09] History of Appeasement: interrupted"
+				`	"Am I interrupting something, Sayari?"`
+					goto interrupt
+					to display
+						not "Hai Reveal [A09] History of Appeasement: interrupted"
+				`	"Teeneep, stop acting like a child!"`
+					goto childish
+					to display
+						not "Hai Reveal [A09] History of Appeasement: interrupted"
+				# The critical questions from the second and third phases of the discussion:
+				`	"What kinds of deals are the Elders making with pirates?"`
+					goto deals
+					to display
+						has "Hai Reveal [A09] History of Appeasement: interrupted"
+						not "Hai Reveal [A09] History of Appeasement: label deals"
+				`	"Now that we know the situation, how do we fix it?"`
+					goto fix
+					to display
+						has "Hai Reveal [A09] History of Appeasement: label deals"
+						not "Hai Reveal [A09] History of Appeasement: label fix"
+				# Optional follow-up questions are next, so you'll get the critical
+				# information first.
+				`	"Are the two of you usually this argumentative?"`
+					goto friends
+					to display
+						has "Hai Reveal [A09] History of Appeasement: interrupted"
+						not "Hai Reveal [A09] History of Appeasement: label friends"
+				`	"Did all of the deals involve arming pirates?"`
+					goto types
+					to display
+						has "Hai Reveal [A09] History of Appeasement: label deals"
+						not "Hai Reveal [A09] History of Appeasement: label types"
+				`	"Does Hai media not deal with this sort of thing?"`
+					goto media
+					to display
+						has "Hai Reveal [A09] History of Appeasement: label fix"
+						not "Hai Reveal [A09] History of Appeasement: label media"
+				# End discussion.
+				`	"We've discussed enough. Let's get to work."`
+					goto go
+					to display
+						has "Hai Reveal [A09] History of Appeasement: label fix"
+
+			# First phase of conversation: listen in or interrupt.
+			label listen
+			action
+				set "Hai Reveal [A09] History of Appeasement: label listen"
+			`	It is difficult to hear precisely without making yourself known, so you struggle to understand. At first, Teeneep is annoyed, maybe saying words like: pirate, victim, orchestra, and betrayal. Sayari's responses revolve around: necessity, an angry child, guilt, and eggs. Something sets off Teeneep, and she keeps screaming "tweezers." Sayari puts a paw gently on Teeneep's shoulders and says in a soothing maternal tone, "green the moon blue." At this point, you realize the conversation is entirely in Hai, which you don't speak, and you have no idea what the two women are actually discussing.`
```

Comment:
@warp-core this is the specific bit I want you to look at

Reply:
Originally, this paragraph was written, so Captain Last knew at the beginning it was in Hai and was trying (and failing) to translate. Someone's suggestion (@EjoThims I think) changed it to not knowing it is in Hai until the end. Either makes sense to me, so if you want to interject on what is best, please do.

Reply:
I'm not really sure why Grey wanted me to look at this specifically. It seems fine, I guess? Though, it's a bit weird that it doesn't occur to the character that they have no reason to be speaking the human language here sooner? It is fun to read, though.
Also, it should probably be `shoulder`, not `shoulders`, if Sayari is only using one paw, unless it's like a really big one.

Reply:
> they have no reason to be speaking the human language here sooner?

It made a lot more sense when Captain Last knew from the beginning that it was in Hai and was trying to translate. That way, though, you get the punchline at the beginning and it isn't as funny.

Reply:
I think, overall, the "fun factor" outweighs the concern, I would change it slightly, though.
```suggestion
			`	It is difficult to hear precisely without making yourself known, so you struggle to understand. At first, Teeneep is annoyed, maybe saying words like: pirate, victim, orchestra, and betrayal. Sayari's responses revolve around: necessity, an angry child, guilt, and eggs. Something sets off Teeneep, and she keeps screaming "tweezers." Sayari puts a paw gently on Teeneep's shoulders and says in a soothing maternal tone, "green the moon blue." At this point, it becomes clear that the conversation is entirely in Hai, which you don't speak, and you have no idea what the two women are actually discussing.`
```

Reply:
That's what I wanted to check, thank you warp